RetailCoder.VBE/Root/RubberduckModule.cs

```diff
@@ -306,6 +306,11 @@ private void BindQuickFixTypes(IEnumerable<Assembly> assemblies)
             foreach (var quickFix in quickFixes)
             {
                 Bind<IQuickFix>().To(quickFix).InSingletonScope();
+                if (quickFix.Name.Equals("AssignedByValParameterMakeLocalCopyQuickFix"))
```

Comment:
@Hosch250 that would be weird. I'm not seeing how an abstract factory interface could escape this binding (regardless of which assembly it's in) if its name follows the `IFooFactory` convention:

````csharp
        // note convention: abstract factory interface names end with "Factory".
        private void ApplyAbstractFactoryConvention(IEnumerable<Assembly> assemblies)
        {
            Kernel.Bind(t => t.From(assemblies)
                .SelectAllInterfaces()
                .Where(type => type.Name.EndsWith("Factory")) 
                .BindToFactory()
                .Configure(binding => binding.InSingletonScope()));
        }
````